server/services/interventionsService.test.ts

```diff
@@ -832,7 +835,6 @@ pactWith({ consumer: 'Interventions UI', provider: 'Interventions Service' }, pr
             {
               id: 'dfb64747-f658-40e0-a827-87b4b0bdcfed',
               createdAt: '2020-12-07T20:45:21.986389Z',
-              createdByUserId: '8751622134',
```

Comment:
this was a backend change. the field is not used on the front end anywhere so was removed.